docker/login.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#
+# Usage: $ ./login.sh
+
+set -eu
+
+docker login -u $DOCKER_LOGIN -p $DOCKER_PASSWORD
```

Comment:
${DOCKER_LOGIN}, ${DOCKER_PASSWORD}

How are these values sourced?

Reply:
We references `login.sh` in [`config.yml`](https://github.com/MarquezProject/marquez/blob/build-tag-push/.circleci/config.yml#L27) as a convenience script. Also, both  `DOCKER_LOGIN` and `DOCKER_PASSWORD` will be set when the CI pipeline kicks off for a release.